packages/ui-components/src/svg/SapSvg.tsx

```diff
@@ -0,0 +1,42 @@
+import React from "react"
+
+interface SapSvgProps {
+  className?: string
+  alt?: string
+}
+
+export const SapSvg: React.FC<SapSvgProps> = ({ className, alt }) => {
```

Comment:
Is this the only way to use SVGs as components in TypeScript? This means we always have to take the extra step of turning every image into a TypeScript component. In addition you can't view them in an image editor or other context where you want to see what the image looks like. I would like to be able to keep the image source files as proper images. If TypeScript requires images to be TypeScript components then can we write some sort of wrapper that takes an image and turns it into a component?

Reply:
I have an idea(how I already solved svgs problems before), we could discuss it on a 1:1 call.